### PR TITLE
Add diagnostics

### PR DESCRIPTION
**Pull request by TimGels**
* Adds diagnostics to get more details about failures with the goal of
  improving stability.
* Added a setting for users to disable the diagnostics.
* Moved LibVLCSharp initialization after component initialization in the
  app.xaml.cs.

To get a visual understanding at the changes on the settings page, I took a screenshot. 
afbeelding

### CODE REVIEW

File: OneDrive-Cloud-Player/App.xaml.cs [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ﻿using LibVLCSharp.Shared;
+using Microsoft.AppCenter;
+using Microsoft.AppCenter.Analytics;
```
Comment (JohannesKauffmann) [inline OneDrive-Cloud-Player/App.xaml.cs:3]: Analytics seems to be imported but not used?

File: OneDrive-Cloud-Player/OneDrive-Cloud-Player.csproj [hunk 2]
```diff
@@ -259,6 +259,12 @@
     <PackageReference Include="LibVLCSharp">
       <Version>3.6.6</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.AppCenter.Analytics">
```
Comment (JohannesKauffmann) [inline OneDrive-Cloud-Player/OneDrive-Cloud-Player.csproj:262]: ditto

File: OneDrive-Cloud-Player/App.xaml.cs (hunk 1 above)
Comment (TimGels) [reply OneDrive-Cloud-Player/App.xaml.cs]: Removed it

**Review by JohannesKauffmann — APPROVED** [review]

**Comment by JohannesKauffmann** [comment]
We still need to figure out how to get the client id into the shipped package, but that's for another PR.

**Pull request merged**